api/net/Areas/Admin/Models/MediaType/MediaTypeModel.cs

```diff
@@ -0,0 +1,76 @@
+using TNO.API.Models;
+using TNO.Entities;
+
+namespace TNO.API.Areas.Admin.Models.MediaType;
+
+/// <summary>
+/// MediaTypeModel class, provides a model that represents an media type.
+/// </summary>
+public class MediaTypeModel : AuditColumnsModel
+{
+    #region Properties
+    /// <summary>
+    /// get/set - The primary key of the type model.
+    /// </summary>
+    public int Id { get; set; } = default!;
+
+    /// <summary>
+    /// get/set - The unique name of the model.
+    /// </summary>
+    public string Name { get; set; } = "";
+
+    /// <summary>
+    /// get/set - A description of the type model.
+    /// </summary>
+    public string Description { get; set; } = "";
+
+    /// <summary>
+    /// get/set - Whether this model is enabled.
+    /// </summary>
+    public bool IsEnabled { get; set; }
+
+    /// <summary>
+    /// get/set - The sort order of the models.
+    /// </summary>
+    public int SortOrder { get; set; }
+    #endregion
+
+    #region Constructors
+    /// <summary>
+    /// Creates a new instance of an MediaTypeModel.
+    /// </summary>
+    public MediaTypeModel() { }
+
+    /// <summary>
+    /// Creates a new instance of an MediaTypeModel, initializes with specified parameter.
+    /// </summary>
+    /// <param name="entity"></param>
+    public MediaTypeModel(Entities.MediaType entity) : base(entity)
+    {
+        this.Id = entity.Id;
+        this.Name = entity.Name;
+        this.Description = entity.Description;
+        this.SortOrder = entity.SortOrder;
+        this.IsEnabled = entity.IsEnabled;
+    }
+    #endregion
+
+    #region Methods
+    /// <summary>
+    /// Explicit conversion to entity.
+    /// </summary>
+    /// <param name="model"></param>
+    public static explicit operator Entities.MediaType(MediaTypeModel model)
+    {
+        var entity = new Entities.MediaType(model.Name)
```

Comment:
Looks like you're missing setting the `Version` property.  Which will result in failures for updates because of concurrency.

Reply:
@Fosol, added Version. Still getting a failure (my whole container dies). Will try to gather some further logs